gizmos/styles.css

```diff
@@ -0,0 +1,50 @@
+body{
```

Comment:
Good job on these css rules! 🎆

One thing to keep in mind is that it's good to keep the indentation consistent as well, try to write your rules in this pattern:

```css
selector {
  first-rule: rule-value;
  second-rule: second-rule-value;
} 
```

Note how there's a space after the selector and before the curly brace and all of the rules are on the same indented level